pages/index.js

```diff
@@ -1,7 +1,63 @@
+import useSWR from "swr";
+import GameCard from "@/components/GameCard";
+import styled from "styled-components";
+import { DYNAMIC_URL } from "../utils/dynamicURLs";
+
+const dynamicUrl = DYNAMIC_URL;
```

Comment:
this renaming seems to be a bit unnecessary. You can use `DYNAMIC_URL` directly in the url for useSWR.

Reply:
done